src/test/java/nl/knaw/dans/pf/language/ddm/api/Ddm2EmdCrosswalkTest.java

```diff
@@ -285,6 +285,30 @@ public void languageWithSchemeAndId() throws Exception {
     }
 
     @Test
+    public void languageWithSchemeAndIdButNoEmdCode() throws Exception {
+        // @formatter:off
+        String ddm = "<?xml version='1.0' encoding='utf-8'?><ddm:DDM"
+            + "  xmlns:dc='http://purl.org/dc/elements/1.1/'"
+            + "  xmlns:ddm='http://easy.dans.knaw.nl/schemas/md/ddm/'"
+            + "  xmlns:dcterms='http://purl.org/dc/terms/'"
+            + "  xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'>"
+            + "  <ddm:dcmiMetadata>"
+            + "    <dc:language xsi:type='dcterms:ISO639-2'>gre</dc:language>"
+            + "  </ddm:dcmiMetadata>"
+            + "</ddm:DDM>";
+        // @formatter:on
+
+        DefaultElement top = firstEmdElementFrom(ddm);
+        assertThat(top.elements().size(), is(1));
+        assertThat(top.getQualifiedName(), is("emd:language"));
+
+        DefaultElement sub = (DefaultElement) top.elements().get(0);
+        assertThat(sub.getQualifiedName(), is("dc:language"));
+        assertThat(sub.getText(), is("gre"));
+        assertThat(sub.attributeCount(), is(0));
```

Comment:
Yes, after this 'fix' the languages values become visible and editable (and deletable, yes, yes). 
Personally checked this on the VM with a fixed deposit. The only downside is that we lose some information because the 'other' codes now end up in a free text field.